fetch.py

```diff
@@ -29,8 +29,20 @@ def get_chat():
 
 
 def get_violations():
+    '''
+    This maps violation properties to a weight which is uses to determine
+    if a violation is 'interesting'. This is completely subjective.
+    '''
+    weight_map = {
+        'alcohol': 1,
+        'belts': 1,
+        'property_damage': 1,
+        'work_zone': 2,
+        'hazmat': 2,
+        'arrest_type': 2,
+    }
     params = {
-        '$limit': 10,
+        '$limit': 50000,
```

Comment:
You should commit the limit you used to generate the geojson file. Ignore if it's actually 50000.